winpr/libwinpr/input/keycode.c

```diff
@@ -878,41 +878,32 @@ DWORD GetVirtualKeyCodeFromKeycode(DWORD keycode, DWORD dwFlags)
 DWORD GetKeycodeFromVirtualKeyCode(DWORD vkcode, DWORD dwFlags)
 {
 	DWORD index;
-	DWORD keycode = 0;
+	DWORD* targetArray;
+	size_t targetSize;
 
-	if (dwFlags & KEYCODE_TYPE_APPLE)
-	{
-		for (index = 0; index < ARRAYSIZE(KEYCODE_TO_VKCODE_APPLE); index++)
-		{
-			if (vkcode == KEYCODE_TO_VKCODE_APPLE[index])
-			{
-				keycode = index;
-				break;
-			}
-		}
-	}
-	else if (dwFlags & KEYCODE_TYPE_EVDEV)
+	switch (dwFlags)
```

Comment:
`dwFlags` could maybe be renamed too.

Reply:
@pnowack pushed a commit for that and the invalid `+ 8` offset for mac keyboards